### PR TITLE
Planets controller created 

### PR DESCRIPTION
**Pull request by Teleri**
with all CRUD actions, routes also updated for those. A temporary view was created for show, but has been deleted now.

I have tested new/create, edit/update, and destroy, they all work within the rails console.

### CODE REVIEW

File: app/controllers/planets_controller.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  end
+
+  private
+
+  def find_planet
+    @planet = Planet.find(params[:id])
+  end
+  
+  def planet_params
+    params.require(:planet).permit(:name, :size, :distance_from_earth, :price_per_day)
```
Comment (mberthier) [inline app/controllers/planets_controller.rb:59]: Should we rename :distance_from_earth to :distance_from_sun?

Comment (Teleri) [reply app/controllers/planets_controller.rb:59]: Yes, I will change the model/schema in a new branch, and also change the controller with that

Comment (mberthier) [reply app/controllers/planets_controller.rb:59]: You need to also add user to a planet. Planet.user = current_user

Comment (Teleri) [reply app/controllers/planets_controller.rb:59]: Also I guess you mean change it to distance_from_sun

**Review by mberthier — COMMENTED** [review]
You need to also add user to a planet. Planet.user = current_user

File: app/controllers/planets_controller.rb (hunk 1 above)
Comment (Teleri) [reply app/controllers/planets_controller.rb:59]: @mberthier Where does this need to go?

Comment (Teleri) [reply app/controllers/planets_controller.rb:59]: Is that not when we have pundit

**Review by mberthier — APPROVED** [review]

**Pull request merged**